string[]  group_names
float64[] time_from_start
---
meka_posture_recorder/PostureRecordErrorCodes error_code
float64 overall_time_from_start
uint32  waypoints
